Bottom: 9caa0ab8ee6e626a40a4c2ed5d5bc4c0c1436e38
Top:    d333c72c78cf04b00a5d66bf1c18701ef394399c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-23 23:43:18 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6e8c5d3..9f1d695 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2698,7 +2698,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
